falcon/http_status.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2013 by Rackspace Hosting, Inc.
```

Comment:
Since you created this file, you can put your own name or the name of your employer (depending on your company's IP policy) here instead of Rackspace.